test/help.test.js

```diff
@@ -45,6 +46,26 @@ describe('loopback generator help', function() {
     process.env.SLC_COMMAND = undefined;
   });
 
+  describe('prints right help message for each generator', function() {
+    var cmdNames = ['acl', 'app', 'boot-script', 'datasource', 'export-api-def',
+    'middleware', 'model', 'property', 'relation', 'remote-method', 'swagger'];
+
+    cmdNames.forEach(function(name) {
+      it('prints right help message for generator ' + name, function() {
+        process.env.SLC_COMMAND = 'slc loopback --help';
```

Comment:
ah..I copied from https://github.com/strongloop/generator-loopback/blob/master/test/help.test.js#L32

Gave a quick look into `strongloop/lib` but still confused how the env works, will debug more about it.

`slc` works fine here. Will change.